index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE=edge">
+      <meta name="viewport" content="width=device-width, initial-scale=1.0">
+      <title>Swiggy</title>
+      <link rel="stylesheet" href="styles/style.css">
+   </head>
+   <body>
+      <header class="header">
+         <img src="images/swiggyLogo.png" alt="Swiggy Logo">
+      </header>
+      <main>
+         <div class="banner">
+            <img src="images/momos.jpg" alt="Momos">
+            <h1>Punjabi Momos</h1>
+         </div>
+         <div class="content">
+            <div class="categories col-4">
+               <ul>
+                  <li class="recommended">Recommended</li>
+                  <li>Chandni Chowk Ki Biryani</li>
+                  <li>Naughty Momos</li>
+                  <li>Naughty Momo Roll</li>
+               </ul>
+            </div>
+            <div class="menu col-4">
+               <h2>Recommended</h2>
+               <p>24 ITEMS</p>
+               <ul>
+                  <li class="dish">
+                     <img src="images/vegMark.png" alt="veg">
+                     <p class="dish-name">Kadhai Paneer Biryani</p>
+                     <p class="price">₹ 249</p>
+                  </li>
+                  <li class="dish">
+                     <img src="images/vegMark.png" alt="veg">
+                     <p class="dish-name">Masala Paneer Biryani</p>
+                     <p class="price">₹ 249</p>
+                  </li>
+               </ul>
+            </div>
+            <div class="cart-wrapper col-4">
+               <div class="cart">
```

Comment:
The indentation is not correct here. Fix the indentation across the file wherever applicable.